src/relion/__init__.py

```diff
@@ -238,3 +240,20 @@ def __exit__(self, exc_type, exc_value, exc_traceback):
         if self.obtained:
             self.lockdir.rmdir()
         self.obtained = False
+
+
+def get_imported(import_job_path):
```

Comment:
I would have expected this to hang off a class. One for future refactoring I suppose

Reply:
Yes, it belongs in the `Project` class. Should be pretty simple for me to fix now